src/clib/Cargo.toml

```diff
@@ -15,3 +15,4 @@ crate-type = ["cdylib"]
 nispor = { path = "../lib" }
 libc = "0.2.74"
 serde_json = "1.0"
+serde_yaml = "0.8.17"
```

Comment:
Yaml is not considered as stable as JSON in python/go.

So as API, we should use json as we do for query.

Reply:
But the input is yaml we have to use the yaml parser, right ?

Reply:
The user of nispor API should convert yaml to json by their own tools. But as C API of nispor, we only support JSON.

There are some bad issue of YAML 1.2 recently fixed, that cause me lose the hope on supporting it as an API schema.